samples/sensor/dps310/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dps310.dts")
```

Comment:
You shouldn't need to set DTC_OVERLAY_FILE.  You can use `app.overlay` for the dts file name and the build system will pick it up for you.  (see samples/sensor/amg88xx for an example).